marklogic-data-hub-central/ui-custom/src/components/DateRangeFacet/DateRangeFacet.tsx

```diff
@@ -0,0 +1,144 @@
+import React, {useContext, useEffect, useState} from "react";
+import "./DateRangeFacet.scss";
+import OverlayTrigger from "react-bootstrap/OverlayTrigger";
+import Tooltip from "react-bootstrap/Tooltip";
+import {InfoCircleFill, Calendar4, XLg} from "react-bootstrap-icons";
+import { SearchContext } from "../../store/SearchContext";
+import moment from "moment";
+import DateRangePicker from "react-bootstrap-daterangepicker";
+import "bootstrap-daterangepicker/daterangepicker.css";
+
+type Props = {
+  data?: any;
+};
+
+const DateRangeFacet: React.FC<Props> = (props) => {
```

Comment:
When I apply the DateRangeFacet to a range, and then remove that range, the range still persist in the calendar when I open the calendar up.

I expect for the calendar range to be cleared when I clear the facet. Can we do that? See this screencast:

https://watch.screencastify.com/v/aAEB0918ZLD6WYxCXV8i